soh/soh/Enhancements/randomizer/3drando/item_location.cpp

```diff
@@ -1011,6 +1011,56 @@ std::vector<std::vector<uint32_t>> ShopLocationLists = {
   GC_ShopLocations,
 };
 
+//List of scrubs, used for pricing the scrubs
+std::vector<uint32_t> ScrubLocations = {
```

Comment:
i kind of wish we didn't have to manually list these here, but it does seem like the most efficient way to handle it at runtime